src/Pim/Bundle/CatalogBundle/Doctrine/Common/Filter/ObjectIdResolver.php

```
@@ -43,7 +44,12 @@ public function getIdsFromCodes($entityName, array $codes)
 
         $ids = [];
         foreach ($codes as $code) {
-            $entity = $repository->findOneBy(['code' => $code]);
+            $criterias = ['code' => $code];
+            if (null !== $attribute) {
```

is it possible ? Maybe it should fail if the attribute is not defined no ?